independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,241 @@
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get latest by country code or zone",
+        "description": "This endpoint retrieves the last known state of a country. It is also possible to get data for a specific zone, in case the country is divided by zones seen on app.electricitymap.org",
+        "operationId": "GetLatestbyCode",
+        "parameters": [
+          {
+            "name": "countryCode",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Country or Zone",
+            "x-ms-summary": "Country or Zone",
+            "x-ms-visibility": "important",
+            "x-ms-dynamic-values": {
+              "operationId": "GetZones",
+              "value-path": "Name",
+              "value-collection": "Zones",
+              "value-title": "Details/zoneName"
+            }
+          }
+        ]
+      }
+    },
+    "/latestbyloc": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "_disclaimer": {
+                  "type": "string",
+                  "description": "Data Disclaimer"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "countryCode": {
+                  "type": "string",
+                  "description": "Country Code"
+                },
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "datetime": {
+                      "type": "string",
+                      "description": "Date/time"
+                    },
+                    "carbonIntensity": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Carbon Intensity"
+                    },
+                    "fossilFuelPercentage": {}
+                  },
+                  "description": "Zone Data"
+                },
+                "units": {
+                  "type": "object",
+                  "properties": {
+                    "carbonIntensity": {
+                      "type": "string",
+                      "description": "Carbon intensity units"
+                    }
+                  },
+                  "description": "Units of measure"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get latest by geographic coordinate",
+        "description": "Get latest by geographic coordinate",
+        "operationId": "GetLatestbyLatLon",
+        "parameters": [
+          {
+            "name": "lon",
+            "in": "query",
+            "required": false,
+            "type": "number",
+			"description" : "Longitude",

```

Comment:
@paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.